5.6/s2i/bin/run

```diff
@@ -20,4 +20,27 @@ export PHP_INI_SCAN_DIR=${PHP_INI_SCAN_DIR:-/etc/opt/rh/rh-php56/php.d}
 envsubst < /opt/app-root/etc/php.ini.template > $PHPRC
 envsubst < /opt/app-root/etc/php.d/10-opcache.ini.template > $PHP_INI_SCAN_DIR/10-opcache.ini
 
+HTTPD_CONFDIR=/opt/rh/httpd24/root/etc/httpd/conf.d
+
+if [ -n "$PHP_OPTIONS" ]; then
+  for opt in $PHP_OPTIONS
```

Comment:
this is your PR and proposal, i'm not going to test your feature for you.  i'm not claiming it won't work, i'm just asking what it looks like so we understand what a user has to do if they need to define an env variable like that, and decide if that is an acceptable user experience.